sns_ik_lib/src/fosns_velocity_ik.cpp

```diff
@@ -30,7 +30,7 @@ using namespace sns_ik;
 
 FOSNSVelocityIK::FOSNSVelocityIK(int dof, Scalar loop_period) :
     FSNSVelocityIK(dof, loop_period),
-    scaleMargin(0.9)
+    scaleMargin(0.98)
```

Comment:
This was set based on the test results and seeing how it performed on the robot. There's a setScaleMargin function if end users have a better value.